controlled-vocabulary/src/main/java/org/uniprot/cv/xdb/UniProtXDbTypes.java

```diff
@@ -19,33 +19,33 @@
     private final String FILENAME = "META-INF/drlineconfiguration.json";
     private final String NEW_DB_LIST = "META-INF/new_db_list.txt";
 
-    private List<UniProtXDbTypeDetail> types = new ArrayList<>();
-    private Map<String, UniProtXDbTypeDetail> typeMap;
+    private List<UniProtDatabaseDetail> types = new ArrayList<>();
+    private Map<String, UniProtDatabaseDetail> typeMap;
 
     UniProtXDbTypes() {
```

Comment:
rename class to UniProtCrossReferenceTypes, or UniProtXRefTypes? or anything else ... I had noticed that there are some classes/methods containing:
... Xref ...
... XRef ...
... XDb ...

And it looks like you've renamed most, which is great